pkg/annotations/ingress_annotations.go

```diff
@@ -0,0 +1,37 @@
+package annotations
+
+import (
+	"k8s.io/api/extensions/v1beta1"
+)
+
+const (
+	// BackendPathPrefixKey defines the value for Path which should be used as a prefix for all HTTP requests.
```

Comment:
Shouldn't the comment be " BackendPathPrefixKey defines the key of the annotation which needs to be set in order to specify the prefix for all HTTP requests" ?  (Key vs Value clarification)

For all appgw annotations that we will add in future, maybe good to create a annotation base definition later on - "appgw.ingress.kubernetes.io"

Reply:
Will create a base annotation when add more.